src/api/srv/converter.ts

```diff
@@ -6,26 +6,26 @@ import { v4 as uuidv4 } from "uuid";
 import { ConverterService as cs } from "./types/entities";
 
 // PARAMETERS FOR AZURE OPENAI SERVICES
-const ENGINE = "SCE_EMEA";
-const MAX_TOKENS = 500;
-const TEMPERATURE = 0.8;
+const MAX_TOKENS = 2000;
+const TEMPERATURE = 0.0;
 const FREQUENCY_PENALTY = 0;
 const PRESENCE_PENALTY = 0;
 const STOP_SEQUENCE: any = null;
 
 const GPT_PARAMS = {
-    engine: ENGINE,
     max_tokens: MAX_TOKENS,
     temperature: TEMPERATURE,
     frequency_penalty: FREQUENCY_PENALTY,
     presence_penalty: PRESENCE_PENALTY,
     stop: STOP_SEQUENCE
 };
```

Comment:
do we still need this?

Reply:
Removed them. Tested locally without them and it worked.